apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/class-full-site-editing.php

```diff
@@ -80,6 +81,25 @@ public static function get_instance() {
 		return self::$instance;
 	}
 
+	/**
+	 * We fire this on priority 98, just before the 99 that the REST API uses
+	 * to translate registered post types into REST routes. We unset the Core
+	 * post_type and set ours.
+	 *
+	 * @return void
+	 */
+	public function ensure_correct_controller() {
+		if ( ! \A8C\FSE\is_full_site_editing_active() ) {
```

Comment:
Is it safe to expect that this function exists in this context?

Reply:
Yup, it's always loaded early in the Editing Toolkit plugin